cmd/geras/main.go

```diff
@@ -31,6 +36,9 @@ import (
 	opentsdb "github.com/G-Research/opentsdb-goclient/client"
 
 	grpc_prometheus "github.com/grpc-ecosystem/go-grpc-prometheus"
+
+	jaegerPropagator "go.opentelemetry.io/contrib/propagators/jaeger"
```

Comment:
I'm disappointed these are two jaegers.. https://blog.golang.org/package-names "Avoid unnecessary package name collisions"

Anyway, vague convention is to use names like `jaeger_propagator` here to make it clear it's a package name rather than a local variable.

Reply:
Fixed